CHANGELOG.md

```diff
@@ -53,8 +53,9 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - [Multi DataSource] Address UX comments on index pattern management stack ([#2611](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2611))
 - [Multi DataSource] Apply get indices error handling in step index pattern ([#2652](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2652))
 - [Vis Builder] Last Updated Timestamp for visbuilder savedobject is getting Generated ([#2628](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2628))
-- Removed Leftover X Pack references ([#2638](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2638)) 
+- Removed Leftover X Pack references ([#2638](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2638))
 - Removes Add Integration button ([#2723](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2723))
+- Change geckodriver version to make consistency ([#2772](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2772))
```

Comment:
Probably a tossup, but I think this goes under security.

Reply:
This is the PR: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2397 that solves CVE. It is merged before we start updating CHANGLOG. This PR is just for consistency purpose which is brought in by a previous PR. Here are more details:
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2771

Reply:
Should we put it in security or bug? @joshuarrrr @AMoo-Miki

Reply:
lol, I changed my mind - since it's an after the fact cleanup, just put it under "Maintenance".